### PR TITLE
multiple fixes in layout

### PR DESCRIPTION
**Pull request by michalzy**

### CODE REVIEW

File: package-lock.json
Comment (alkeicam) [inline package-lock.json:1]: please do not put package-lock.json into git

File: src/css/style.css [hunk 1]
```diff
@@ -1,15 +1,99 @@
-.is-vis-hidden{
-    visibility: hidden;
+.is-vis-hidden {
```
Comment (alkeicam) [inline src/css/style.css]: use https://bulma.io/documentation/helpers/visibility-helpers/ instead

File: src/js/rivet-ext.js [hunk 2]
```diff
@@ -116,6 +116,10 @@
         return false;
     }
 
+    rivets.formatters.not = function(value) {
```
Comment (alkeicam) [inline src/js/rivet-ext.js]: please user formatters.eq instead

File: src/index.html [hunk 3]
```diff
@@ -329,187 +331,188 @@ <h4 class="title is-4 has-text-weight-bold">
...
-                </div> -->
-      <!-- <h3 class="title is-4">Trends & Dynamics</h3>
-      <div data-kind="usersMa" data-effort-key="cals" data-title="User Contribution"
-        rv-plot-stats="model.trends.all_time"></div>
-      <div data-kind="usersMa" data-effort-key="entropy" data-title="User Efficiency"
-        rv-plot-stats="model.trends.all_time"></div> -->
-      <!-- <app-stats-plot kind="users" effort-key="cals" title="Daily User Calories" stats="model.trends.all_time"></app-stats-plot> -->
-      <!-- </section> -->
+    </div>
+    <div class="loader-wrapper" rv-class-is-hidden="model.busy | not">
```
Comment (alkeicam) [inline src/index.html]: use | eq false instead

**Review by alkeicam — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (michalzy) [reply package-lock.json:1]: @alkeicam done

File: src/css/style.css (hunk 1 above)
Comment (michalzy) [reply src/css/style.css]: @alkeicam  class was not used anywhere, deleted

File: src/js/rivet-ext.js (hunk 2 above)
Comment (michalzy) [reply src/js/rivet-ext.js]: @alkeicam removed

File: src/index.html (hunk 3 above)
Comment (michalzy) [reply src/index.html]: @alkeicam done

**Pull request merged**